src/components/Item/index.js

```diff
@@ -58,7 +58,7 @@ class Item extends Component {
         <p>{title}</p>
         <img className="img-item" src={thumbnail} alt={title} />
         <p>R${price}</p>
-        <p>{shipping.free_shipping && 'Free'}</p>
+        {shipping.free_shipping && <p data-testid="free-shipping">Free</p>}
```

Comment:
Vai se ferrar.
```suggestion
        {shipping.free_shipping && <p data-testid="free-shipping">Free</p>}
```
Que tal?